.github/dependabot.yml

```diff
@@ -1,36 +1,32 @@
-# Please see the documentation for all configuration options:
-# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# Please see the documentation for all configuration options: https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
 version: 2
 updates:
-# Dependencies listed in go.mod
-- package-ecosystem: "gomod"
-  directory: "/" # Location of package manifests
+# github-actions
+- directory: "/"
+  package-ecosystem: "github-actions"
   schedule:
-    interval: "daily" ## change it to interval:"weekly" once Dependabot starts working. 
-    # day: "monday"
+    interval: "daily"
   commit-message:
-    include: "scope"
-  ignore:
-    # Ignore controller-runtime as its upgraded manually.
-    - dependency-name: "sigs.k8s.io/controller-runtime"
-    # Ignore k8s and its transitives modules as they are upgraded manually
-    # together with controller-runtime.
-    - dependency-name: "k8s.io/*"
-    - dependency-name: "go.etcd.io/*"
-    - dependency-name: "google.golang.org/grpc"
+    include: scope
   labels:
     - "ok-to-test"
     - "kind/cleanup"
-    - "priority/important-soon"
-    - "wg/security-audit"
```

Comment:
Removed 
- `"priority/important-soon"`
- `"wg/security-audit"`
As dependabot.yml is used to configure version updates of dependencies. 


Security updates is not controlled by dependabot.yml config.